website/versioned_docs/version-4.x/auth-flow.md

```diff
@@ -84,7 +83,7 @@ class AuthLoadingScreen extends React.Component {
 }
 ```
 
-<a href="https://snack.expo.io/@react-navigation/auth-flow-v3" target="blank" class="run-code-button">&rarr; Run this code</a>
+<samp id="auth-flow">Auth Flow</samp>
```

Comment:
```suggestion
```

I would leave the snack only for the last example. @satya164 what you think?

Reply:
Sounds good 👍

Reply:
Done, can you check again @WoLewicki  ?

Reply:
It's ok now. Thanks for your contribution!